test/k8sT/manifests/echo-policy.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: "cilium.io/v2"
+kind: CiliumNetworkPolicy
+metadata:
+  name: "allow-all-within-namespace"
+specs:
+  endpointSelector:
```

Comment:
`specs` is for when you want to define multiple policies. I think you want just `spec` here.

Reply:
Yup. Fixed.